tic-tac-toe.js

```diff
@@ -1,7 +1,36 @@
-function TicTacToe() {
+var TicTacToe = function() {
+  this.markers = ["X", "O", "X", "O", "X", "O", "X", "O", "X", "O"];
+  this.i = 0;
+  this.xScore = {A: 0, B: 0, C: 0, D: 0, E: 0, F: 0, G: 0, H: 0};
+  this.oScore = {A: 0, B: 0, C: 0, D: 0, E: 0, F: 0, G: 0, H: 0};
+  this.winner = "Tic Tac Toe";
+};
 
-}
+// prototype that iterated through the markers
+TicTacToe.prototype.mark = function (changeSquare) {
+  var symbol = this.markers[this.i];
+  $(changeSquare).html(symbol);
+};
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.score = function (squaresArray) {
+  // use the appropriate score hash
+  if (this.i % 2 == 0) {
+    var scoreChart = this.xScore;
+  } else {
+    var scoreChart = this.oScore;
+  }
+  // in the score hash add one to the value of the key that corresponds to the letter coming in from the squaresArray
+  var gameThis = this;
```

Comment:
Compliment time: All of your variables have been clearly and concisely named.   gameThis is a little awkward and could be more clearly named.